src/node.cc

```diff
@@ -4383,8 +4384,12 @@ static void StartNodeInstance(void* arg) {
         ShouldAbortOnUncaughtException);
 
     // Start debug agent when argv has --debug
-    if (instance_data->use_debug_agent())
+    if (instance_data->use_debug_agent()) {
       StartDebug(&env, debug_wait_connect);
+      if (use_inspector && !debugger_running) {
+        exit(14);
```

Comment:
Can you document this in doc/api/process.md in the section 'Exit Codes'?  If you can reuse an existing documented exit code, even better.